# ETHZ Block Course- Practical 376-1346-00 Study of Epigenetic Mechanisms in Mental Health 

## Project 6 (Block Course 2021)

### Investigating differential exon usage and differential 3’ untranslated regions in spermatogonial cells across development

Throughout an individual’s life, health is influenced by personal and social experiences, and by various environmental factors. Previous work from our laboratory demonstrated that trauma exposure causes behavioural, physiological and molecular alterations that can be transmitted to descendants across up to four generations in mice. Transmission is germline-dependent and involves epigenetic and transcriptional changes in germ cells. Recent analyses revealed that RNA in spermatogonial cells (SCs), germ stem cells in testis that initiate and support spermatogenesis in mammals, is altered by exposure, and in particular that alternative splicing is perturbed. Alternative splicing is a process that enables the production of various forms of messenger RNA (mRNA) from a given DNA sequence to direct the synthesis of different protein variants (isoforms). Isoforms can have different cellular functions and properties, thus alternative splicing greatly increases the functional diversity of genes.

In the context of RNA-sequencing analysis, alternative splicing presents many challenges. Bioinformatics approaches have been developed to analyze its features such as differential transcript expression, which is the difference in expression of individual transcripts between groups, differential transcript usage, which looks at equally expressed genes that show variable isoform abundance; and differential exon usage, which measures differential relative usage of a particular exon within a gene.

This project will examine features of alternative splicing in a set of RNA-sequencing data collected from SCs from males exposed to postnatal trauma (MSUS) at three developmental stages: postnatal day 8 (PND8), PND15 and postnatal week 20 (PNW20). PND8 and PND15 are two key time points of spermatogenesis that correspond to direct exposure to trauma, while the adult time point is many month after exposure. Using these datasets with a higher number of replicates and high-quality, we will investigate changes at the level of genes, transcripts, exons and also looked into differential isoform usage. This project will explore differential exon usage and differential 3’ untranslated regions using a new tool, diffUTR, developed by Dr. Pierre-Luc Germain. This project will also involve an essential component of exploratory data analysis and a proper graphical representation of results.
For this project, prior experience of working with R is welcome but not essential. Introduction to programming with R, theoretical background on library preparation methods, quality check and quality control of sequenced data, alignment, and differential analysis will be provided during the first week.

**Methods:**

-	Differential exon usage
-	Differential UTRs analysis
-	Exploratory data analysis
-	Interpretation of the results in relevance to the MSUS
-	GitHub for project organization
-	The final presentation of the work


| Name | Role | Email | GitHub username|
| -- | -- | -- | -- |
| Deepak Tanwar | Supervisor | [tanwar@hifo.uzh.ch](mailto:tanwar@hifo.uzh.ch) | [@dktanwar](https://github.com/dktanwar) |
| Dr. Pierre-Luc Germain | Supervisor | [germain@hifo.uzh.ch](mailto:germain@hifo.uzh.ch) | [@plger](https://github.com/plger) |
| David Bugliani | Student | [bugdavid@student.ethz.ch](bugdavid@student.ethz.ch) | [@BuglianiD](https://github.com/BuglianiD) |
| Julien Chabbey | Student | [chabbeyj@student.ethz.ch](mailto:chabbeyj@student.ethz.ch) | [@Chabbeyj](https://github.com/Chabbeyj) |
