<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Differential Analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">project6_bc2021</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mansuylab/project6_bc2021">
    <span class="fas fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Differential Analysis</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-04-11
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>project6_bc2021/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div class="panel-group" id="workflowr-checks">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210323code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210323)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210323code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210323)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021
</td>
<td style="text-align:left;">
.
</td>
</tr>
<tr>
<td style="text-align:left;">
C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021/data/counts_bins_apa.rds
</td>
<td style="text-align:left;">
data/counts_bins_apa.rds
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommansuylabproject6bc2021tree000b69c45042d48b03c1d595522828411333e894targetblank000b69ca"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/mansuylab/project6_bc2021/tree/000b69c45042d48b03c1d595522828411333e894" target="_blank">000b69c</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcommansuylabproject6bc2021tree000b69c45042d48b03c1d595522828411333e894targetblank000b69ca" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/mansuylab/project6_bc2021/tree/000b69c45042d48b03c1d595522828411333e894" target="_blank">000b69c</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    data/counts_bins.rds

Untracked files:
    Untracked:  data/counts_bins_apa.rds

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/2_Differential_Analysis_JC.Rmd</code>) and HTML (<code>docs/2_Differential_Analysis_JC.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mansuylab/project6_bc2021/blob/000b69c45042d48b03c1d595522828411333e894/analysis/2_Differential_Analysis_JC.Rmd" target="_blank">000b69c</a>
</td>
<td>
Julien Chabbey
</td>
<td>
2021-04-11
</td>
<td>
graph for top 10 genes
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mansuylab/project6_bc2021/blob/fedf7ca62c9e16ba7b51151be3ce5c1072e0fe6a/analysis/2_Differential_Analysis_JC.Rmd" target="_blank">fedf7ca</a>
</td>
<td>
Julien Chabbey
</td>
<td>
2021-03-31
</td>
<td>
html files and relocation
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mansuylab/project6_bc2021/fedf7ca62c9e16ba7b51151be3ce5c1072e0fe6a/docs/2_Differential_Analysis_JC.html" target="_blank">fedf7ca</a>
</td>
<td>
Julien Chabbey
</td>
<td>
2021-03-31
</td>
<td>
html files and relocation
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>#Library</p>
<pre class="r"><code>library(diffUTR)
library(SummarizedExperiment)
library(GenomicRanges)
library(dplyr)</code></pre>
<p>#working directories and data</p>
<pre class="r"><code>setwd(&quot;C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021&quot;)
se &lt;- readRDS(&quot;C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021/data/counts_bins_apa.rds&quot;)

se_apa &lt;- readRDS(&quot;C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021/data/counts_bins_apa.rds&quot;)</code></pre>
<p>#subset PND8</p>
<pre class="r"><code>cd &lt;- data.frame(colData(se), stringsAsFactors = F)
pnd8 &lt;- cd$Stage == &quot;PND8&quot;
sePND8 &lt;- se[,pnd8]</code></pre>
<p>#Exon analysis</p>
<pre class="r"><code>sePND8_exon &lt;- sePND8
sePND8_exon &lt;- diffSpliceWrapper(sePND8_exon, design = ~Batch+Group)</code></pre>
<pre><code>Testing coefficient GroupMSUS</code></pre>
<pre><code>Total number of exons:  316430 
Total number of genes:  22903 
Number of genes with 1 exon:  5745 
Mean number of exons in a gene:  14 
Max number of exons in a gene:  418 </code></pre>
<pre class="r"><code>head(rowData(sePND8_exon))</code></pre>
<pre><code>DataFrame with 6 rows and 28 columns
                              source     type         score         phase
                        &lt;FactorList&gt; &lt;factor&gt; &lt;NumericList&gt; &lt;IntegerList&gt;
ENSMUSG00000000001.4.1        HAVANA     3UTR                            
ENSMUSG00000000001.4.10       HAVANA     3UTR                            
ENSMUSG00000000001.4.11       HAVANA     3UTR                            
ENSMUSG00000000001.4.12       HAVANA     3UTR                            
ENSMUSG00000000001.4.13       HAVANA     3UTR                            
ENSMUSG00000000001.4.14       HAVANA     3UTR                            
                              gene_name           level          havana_gene
                        &lt;CharacterList&gt; &lt;CharacterList&gt;      &lt;CharacterList&gt;
ENSMUSG00000000001.4.1            Gnai3               2 OTTMUSG00000007197.1
ENSMUSG00000000001.4.10           Gnai3               2 OTTMUSG00000007197.1
ENSMUSG00000000001.4.11           Gnai3               2 OTTMUSG00000007197.1
ENSMUSG00000000001.4.12           Gnai3               2 OTTMUSG00000007197.1
ENSMUSG00000000001.4.13           Gnai3               2 OTTMUSG00000007197.1
ENSMUSG00000000001.4.14           Gnai3               2 OTTMUSG00000007197.1
                         transcript_name transcript_support_level
                         &lt;CharacterList&gt;          &lt;CharacterList&gt;
ENSMUSG00000000001.4.1  RP23-89M15.4-001                        1
ENSMUSG00000000001.4.10 RP23-89M15.4-001                        1
ENSMUSG00000000001.4.11 RP23-89M15.4-001                        1
ENSMUSG00000000001.4.12 RP23-89M15.4-001                        1
ENSMUSG00000000001.4.13 RP23-89M15.4-001                        1
ENSMUSG00000000001.4.14 RP23-89M15.4-001                        1
                                    tag    havana_transcript     exon_number
                        &lt;CharacterList&gt;      &lt;CharacterList&gt; &lt;CharacterList&gt;
ENSMUSG00000000001.4.1             CCDS OTTMUST00000016610.1               9
ENSMUSG00000000001.4.10            CCDS OTTMUST00000016610.1             8,9
ENSMUSG00000000001.4.11            CCDS OTTMUST00000016610.1             8,9
ENSMUSG00000000001.4.12            CCDS OTTMUST00000016610.1             8,9
ENSMUSG00000000001.4.13            CCDS OTTMUST00000016610.1             8,9
ENSMUSG00000000001.4.14            CCDS OTTMUST00000016610.1             8,9
                                                          exon_id
                                                  &lt;CharacterList&gt;
ENSMUSG00000000001.4.1                       ENSMUSE00000363317.2
ENSMUSG00000000001.4.10 ENSMUSE00000363317.2,ENSMUSE00000404895.1
ENSMUSG00000000001.4.11 ENSMUSE00000363317.2,ENSMUSE00000404895.1
ENSMUSG00000000001.4.12 ENSMUSE00000363317.2,ENSMUSE00000404895.1
ENSMUSG00000000001.4.13 ENSMUSE00000363317.2,ENSMUSE00000404895.1
ENSMUSG00000000001.4.14 ENSMUSE00000363317.2,ENSMUSE00000404895.1
                                  protein_id          ccdsid             ont
                             &lt;CharacterList&gt; &lt;CharacterList&gt; &lt;CharacterList&gt;
ENSMUSG00000000001.4.1  ENSMUSP00000000001.4     CCDS17751.1                
ENSMUSG00000000001.4.10 ENSMUSP00000000001.4     CCDS17751.1                
ENSMUSG00000000001.4.11 ENSMUSP00000000001.4     CCDS17751.1                
ENSMUSG00000000001.4.12 ENSMUSP00000000001.4     CCDS17751.1                
ENSMUSG00000000001.4.13 ENSMUSP00000000001.4     CCDS17751.1                
ENSMUSG00000000001.4.14 ENSMUSP00000000001.4     CCDS17751.1                
                                          tx                 gene
                                    &lt;factor&gt;             &lt;factor&gt;
ENSMUSG00000000001.4.1  ENSMUST00000000001.4 ENSMUSG00000000001.4
ENSMUSG00000000001.4.10 ENSMUST00000000001.4 ENSMUSG00000000001.4
ENSMUSG00000000001.4.11 ENSMUST00000000001.4 ENSMUSG00000000001.4
ENSMUSG00000000001.4.12 ENSMUST00000000001.4 ENSMUSG00000000001.4
ENSMUSG00000000001.4.13 ENSMUST00000000001.4 ENSMUSG00000000001.4
ENSMUSG00000000001.4.14 ENSMUST00000000001.4 ENSMUSG00000000001.4
                            tx_biotype   gene_biotype      bin_id geneAmbiguous
                          &lt;FactorList&gt;   &lt;FactorList&gt; &lt;character&gt;     &lt;logical&gt;
ENSMUSG00000000001.4.1  protein_coding protein_coding           1         FALSE
ENSMUSG00000000001.4.10 protein_coding protein_coding          10         FALSE
ENSMUSG00000000001.4.11 protein_coding protein_coding          11         FALSE
ENSMUSG00000000001.4.12 protein_coding protein_coding          12         FALSE
ENSMUSG00000000001.4.13 protein_coding protein_coding          13         FALSE
ENSMUSG00000000001.4.14 protein_coding protein_coding          14         FALSE
                        meanLogCPM  logWidth meanLogDensity coefficient
                         &lt;numeric&gt; &lt;numeric&gt;      &lt;numeric&gt;   &lt;numeric&gt;
ENSMUSG00000000001.4.1    0.200155   8.08240    0.000385471   0.9358602
ENSMUSG00000000001.4.10   1.169463   4.53260    0.035159455  -0.1721563
ENSMUSG00000000001.4.11   1.156828   4.53260    0.034731842  -0.1133094
ENSMUSG00000000001.4.12   0.577740   4.36945    0.025253718   0.3897368
ENSMUSG00000000001.4.13   0.886719   5.11799    0.016135681  -0.1073859
ENSMUSG00000000001.4.14   0.501076   3.49651    0.034217185  -0.0856279
                        bin.p.value   bin.FDR
                          &lt;numeric&gt; &lt;numeric&gt;
ENSMUSG00000000001.4.1    0.0040525         1
ENSMUSG00000000001.4.10   0.2203402         1
ENSMUSG00000000001.4.11   0.4200600         1
ENSMUSG00000000001.4.12   0.1878832         1
ENSMUSG00000000001.4.13   0.6043645         1
ENSMUSG00000000001.4.14   0.7879538         1</code></pre>
<pre class="r"><code>perGene &lt;- metadata(sePND8_exon)$geneLevel
head(perGene)</code></pre>
<pre><code>DataFrame with 6 rows and 11 columns
                              name    w.coef w.abs.coef   w.width w.density
                       &lt;character&gt; &lt;numeric&gt;  &lt;numeric&gt; &lt;integer&gt; &lt;numeric&gt;
ENSMUSG00000030216.12        Wbp11     1.296      1.356       114  -2.82068
ENSMUSG00000083626.1    AC141643.2    -1.678      1.989       438  -3.35522
ENSMUSG00000068466.5        Gm5518    -0.454      0.728       625  -2.81937
ENSMUSG00000063800.5       Prpf38a     0.991      1.120       251  -3.63308
ENSMUSG00000081992.1  RP23-14N22.2     0.925      1.056       753  -3.10948
ENSMUSG00000031948.14         Kars     0.930      0.979       140  -2.84797
                      sizeScore abs.sizeScore geneMeanDensity density.ratio
                      &lt;numeric&gt;     &lt;numeric&gt;       &lt;numeric&gt;     &lt;numeric&gt;
ENSMUSG00000030216.12     0.973         1.085           0.038         0.019
ENSMUSG00000083626.1     -1.931         1.989           0.029         0.006
ENSMUSG00000068466.5     -0.497         0.728           0.039         0.020
ENSMUSG00000063800.5      0.723         0.999           0.030        -0.004
ENSMUSG00000081992.1      1.045         1.056           0.032         0.012
ENSMUSG00000031948.14     0.769         0.825           0.047         0.011
                        nb.bins     q.value
                      &lt;integer&gt;   &lt;numeric&gt;
ENSMUSG00000030216.12        33 7.46523e-29
ENSMUSG00000083626.1          2 7.76782e-24
ENSMUSG00000068466.5          2 5.95962e-21
ENSMUSG00000063800.5         23 5.99881e-21
ENSMUSG00000081992.1          2 1.00444e-18
ENSMUSG00000031948.14        29 1.40894e-17</code></pre>
<p>#UTR Analysis</p>
<pre class="r"><code>#subset
cd2 &lt;- data.frame(colData(se_apa))
pnd8_2 &lt;- cd2$Stage == &quot;PND8&quot;
sePND8UTR &lt;- se_apa[,pnd8_2]

#analysis
sePND8_utr &lt;- diffSpliceWrapper(sePND8UTR, design = ~Batch+Group, 
                excludeTypes = c(&quot;CDS&quot;,&quot;non-coding&quot;))</code></pre>
<pre><code>Testing coefficient GroupMSUS</code></pre>
<pre><code>Total number of exons:  316430 
Total number of genes:  22903 
Number of genes with 1 exon:  5745 
Mean number of exons in a gene:  14 
Max number of exons in a gene:  418 </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.4 (2021-02-15)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=French_Switzerland.1252  LC_CTYPE=French_Switzerland.1252   
[3] LC_MONETARY=French_Switzerland.1252 LC_NUMERIC=C                       
[5] LC_TIME=French_Switzerland.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] dplyr_1.0.5                 SummarizedExperiment_1.20.0
 [3] Biobase_2.50.0              GenomicRanges_1.42.0       
 [5] GenomeInfoDb_1.26.4         IRanges_2.24.1             
 [7] S4Vectors_0.28.1            BiocGenerics_0.36.0        
 [9] MatrixGenerics_1.2.1        matrixStats_0.58.0         
[11] diffUTR_0.99.24             workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] colorspace_2.0-0         rjson_0.2.20             hwriter_1.3.2           
  [4] ellipsis_0.3.1           rprojroot_2.0.2          circlize_0.4.12         
  [7] XVector_0.30.0           GlobalOptions_0.1.2      fs_1.5.0                
 [10] clue_0.3-58              ggrepel_0.9.1            bit64_4.0.5             
 [13] AnnotationDbi_1.52.0     fansi_0.4.2              xml2_1.3.2              
 [16] splines_4.0.4            cachem_1.0.4             geneplotter_1.68.0      
 [19] knitr_1.31               jsonlite_1.7.2           Rsamtools_2.6.0         
 [22] Cairo_1.5-12.2           annotate_1.68.0          cluster_2.1.1           
 [25] dbplyr_2.1.0             png_0.1-7                compiler_4.0.4          
 [28] httr_1.4.2               lazyeval_0.2.2           assertthat_0.2.1        
 [31] Matrix_1.3-2             fastmap_1.1.0            limma_3.46.0            
 [34] later_1.1.0.1            htmltools_0.5.1.1        prettyunits_1.1.1       
 [37] tools_4.0.4              gtable_0.3.0             glue_1.4.2              
 [40] GenomeInfoDbData_1.2.4   rappdirs_0.3.3           Rcpp_1.0.6              
 [43] jquerylib_0.1.3          vctrs_0.3.6              Biostrings_2.58.0       
 [46] rtracklayer_1.49.5       xfun_0.22                stringr_1.4.0           
 [49] lifecycle_1.0.0          ensembldb_2.14.0         statmod_1.4.35          
 [52] XML_3.99-0.6             DEXSeq_1.36.0            edgeR_3.32.1            
 [55] zlibbioc_1.36.0          scales_1.1.1             ProtGenerics_1.22.0     
 [58] hms_1.0.0                promises_1.2.0.1         AnnotationFilter_1.14.0 
 [61] RColorBrewer_1.1-2       ComplexHeatmap_2.6.2     yaml_2.2.1              
 [64] curl_4.3                 memoise_2.0.0            ggplot2_3.3.3           
 [67] sass_0.3.1               biomaRt_2.46.3           stringi_1.5.3           
 [70] RSQLite_2.2.4            highr_0.8                genefilter_1.72.1       
 [73] GenomicFeatures_1.42.2   Rsubread_2.4.3           BiocParallel_1.24.1     
 [76] shape_1.4.5              rlang_0.4.10             pkgconfig_2.0.3         
 [79] bitops_1.0-6             evaluate_0.14            lattice_0.20-41         
 [82] purrr_0.3.4              GenomicAlignments_1.26.0 bit_4.0.4               
 [85] tidyselect_1.1.0         magrittr_2.0.1           DESeq2_1.30.1           
 [88] R6_2.5.0                 generics_0.1.0           DelayedArray_0.16.2     
 [91] DBI_1.1.1                pillar_1.5.1             whisker_0.4             
 [94] survival_3.2-10          RCurl_1.98-1.3           tibble_3.1.0            
 [97] crayon_1.4.1             utf8_1.2.1               BiocFileCache_1.14.0    
[100] rmarkdown_2.7            GetoptLong_1.0.5         progress_1.2.2          
[103] locfit_1.5-9.4           grid_4.0.4               blob_1.2.1              
[106] git2r_0.28.0             digest_0.6.27            xtable_1.8-4            
[109] httpuv_1.5.5             openssl_1.4.3            munsell_0.5.0           
[112] bslib_0.2.4              askpass_1.1             </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
