<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Processed Plot</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">project6_bc2021</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mansuylab/project6_bc2021">
    <span class="fas fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Processed Plot</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-03-31
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>project6_bc2021/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div class="panel-group" id="workflowr-checks">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210323code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210323)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210323code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210323)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021
</td>
<td style="text-align:left;">
.
</td>
</tr>
<tr>
<td style="text-align:left;">
C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021/data/counts_bins_apa.rds
</td>
<td style="text-align:left;">
data/counts_bins_apa.rds
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommansuylabproject6bc2021tree7511b10b8a8caeb79887c619de6c2b342867a404targetblank7511b10a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/mansuylab/project6_bc2021/tree/7511b10b8a8caeb79887c619de6c2b342867a404" target="_blank">7511b10</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcommansuylabproject6bc2021tree7511b10b8a8caeb79887c619de6c2b342867a404targetblank7511b10a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/mansuylab/project6_bc2021/tree/7511b10b8a8caeb79887c619de6c2b342867a404" target="_blank">7511b10</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory

Untracked files:
    Untracked:  analysis/1_Preprocessed_plot_JC.Rmd
    Untracked:  analysis/2_Differential_Analysis_JC.Rmd
    Untracked:  analysis/3_Processed_plot_PND8_JC.Rmd
    Untracked:  analysis/4_Vignette_Plot_PND8_JC.Rmd
    Untracked:  apa.mm38.bed.gz
    Untracked:  data/counts_bins.rds
    Untracked:  data/counts_bins_apa.rds

Unstaged changes:
    Deleted:    code/Differential Analysis.Rmd
    Deleted:    code/Differential Analysis.nb.html
    Deleted:    code/Preprocessed plot.Rmd
    Deleted:    code/Preprocessed plot.nb.html
    Deleted:    code/Processed plot - PND8.Rmd
    Deleted:    code/Processed plot - PND8.nb.html
    Deleted:    code/Vignette Plot - PND8.Rmd
    Deleted:    code/Vignette Plot - PND8.nb.html

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<p>#Library</p>
<pre class="r"><code>library(diffUTR)
library(SummarizedExperiment)</code></pre>
<pre><code>Le chargement a nécessité le package : MatrixGenerics</code></pre>
<pre><code>Le chargement a nécessité le package : matrixStats</code></pre>
<pre><code>
Attachement du package : &#39;MatrixGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:matrixStats&#39;:

    colAlls, colAnyNAs, colAnys, colAvgsPerRowSet, colCollapse,
    colCounts, colCummaxs, colCummins, colCumprods, colCumsums,
    colDiffs, colIQRDiffs, colIQRs, colLogSumExps, colMadDiffs,
    colMads, colMaxs, colMeans2, colMedians, colMins, colOrderStats,
    colProds, colQuantiles, colRanges, colRanks, colSdDiffs, colSds,
    colSums2, colTabulates, colVarDiffs, colVars, colWeightedMads,
    colWeightedMeans, colWeightedMedians, colWeightedSds,
    colWeightedVars, rowAlls, rowAnyNAs, rowAnys, rowAvgsPerColSet,
    rowCollapse, rowCounts, rowCummaxs, rowCummins, rowCumprods,
    rowCumsums, rowDiffs, rowIQRDiffs, rowIQRs, rowLogSumExps,
    rowMadDiffs, rowMads, rowMaxs, rowMeans2, rowMedians, rowMins,
    rowOrderStats, rowProds, rowQuantiles, rowRanges, rowRanks,
    rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
    rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
    rowWeightedSds, rowWeightedVars</code></pre>
<pre><code>Le chargement a nécessité le package : GenomicRanges</code></pre>
<pre><code>Le chargement a nécessité le package : stats4</code></pre>
<pre><code>Le chargement a nécessité le package : BiocGenerics</code></pre>
<pre><code>Le chargement a nécessité le package : parallel</code></pre>
<pre><code>
Attachement du package : &#39;BiocGenerics&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:parallel&#39;:

    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    clusterExport, clusterMap, parApply, parCapply, parLapply,
    parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    IQR, mad, sd, var, xtabs</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    anyDuplicated, append, as.data.frame, basename, cbind, colnames,
    dirname, do.call, duplicated, eval, evalq, Filter, Find, get, grep,
    grepl, intersect, is.unsorted, lapply, Map, mapply, match, mget,
    order, paste, pmax, pmax.int, pmin, pmin.int, Position, rank,
    rbind, Reduce, rownames, sapply, setdiff, sort, table, tapply,
    union, unique, unsplit, which.max, which.min</code></pre>
<pre><code>Le chargement a nécessité le package : S4Vectors</code></pre>
<pre><code>
Attachement du package : &#39;S4Vectors&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    expand.grid</code></pre>
<pre><code>Le chargement a nécessité le package : IRanges</code></pre>
<pre><code>
Attachement du package : &#39;IRanges&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:grDevices&#39;:

    windows</code></pre>
<pre><code>Le chargement a nécessité le package : GenomeInfoDb</code></pre>
<pre><code>Le chargement a nécessité le package : Biobase</code></pre>
<pre><code>Welcome to Bioconductor

    Vignettes contain introductory material; view with
    &#39;browseVignettes()&#39;. To cite Bioconductor, see
    &#39;citation(&quot;Biobase&quot;)&#39;, and for packages &#39;citation(&quot;pkgname&quot;)&#39;.</code></pre>
<pre><code>
Attachement du package : &#39;Biobase&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:MatrixGenerics&#39;:

    rowMedians</code></pre>
<pre><code>The following objects are masked from &#39;package:matrixStats&#39;:

    anyMissing, rowMedians</code></pre>
<pre class="r"><code>library(GenomicRanges)
library(dplyr)</code></pre>
<pre><code>
Attachement du package : &#39;dplyr&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:Biobase&#39;:

    combine</code></pre>
<pre><code>The following objects are masked from &#39;package:GenomicRanges&#39;:

    intersect, setdiff, union</code></pre>
<pre><code>The following object is masked from &#39;package:GenomeInfoDb&#39;:

    intersect</code></pre>
<pre><code>The following objects are masked from &#39;package:IRanges&#39;:

    collapse, desc, intersect, setdiff, slice, union</code></pre>
<pre><code>The following objects are masked from &#39;package:S4Vectors&#39;:

    first, intersect, rename, setdiff, setequal, union</code></pre>
<pre><code>The following objects are masked from &#39;package:BiocGenerics&#39;:

    combine, intersect, setdiff, union</code></pre>
<pre><code>The following object is masked from &#39;package:matrixStats&#39;:

    count</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    filter, lag</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    intersect, setdiff, setequal, union</code></pre>
<pre class="r"><code>library(ggplot2)
library(reshape2)
library(limma)</code></pre>
<pre><code>
Attachement du package : &#39;limma&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:BiocGenerics&#39;:

    plotMA</code></pre>
<pre class="r"><code>library(edgeR)
library(ensembldb)</code></pre>
<pre><code>Le chargement a nécessité le package : GenomicFeatures</code></pre>
<pre><code>Le chargement a nécessité le package : AnnotationDbi</code></pre>
<pre><code>
Attachement du package : &#39;AnnotationDbi&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    select</code></pre>
<pre><code>Le chargement a nécessité le package : AnnotationFilter</code></pre>
<pre><code>
Attachement du package : &#39;ensembldb&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    filter</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    filter</code></pre>
<pre class="r"><code>library(ggpubr)
library(SEtools)</code></pre>
<pre><code>Warning: replacing previous import &#39;ComplexHeatmap::pheatmap&#39; by
&#39;pheatmap::pheatmap&#39; when loading &#39;SEtools&#39;</code></pre>
<pre class="r"><code>library(EnhancedVolcano)</code></pre>
<pre><code>Le chargement a nécessité le package : ggrepel</code></pre>
<pre><code>Registered S3 methods overwritten by &#39;ggalt&#39;:
  method                  from   
  grid.draw.absoluteGrob  ggplot2
  grobHeight.absoluteGrob ggplot2
  grobWidth.absoluteGrob  ggplot2
  grobX.absoluteGrob      ggplot2
  grobY.absoluteGrob      ggplot2</code></pre>
<p>#Working directories and data</p>
<pre class="r"><code>setwd(&quot;C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021&quot;)
se &lt;- readRDS(&quot;C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021/data/counts_bins_apa.rds&quot;)

cd &lt;- data.frame(colData(se), stringsAsFactors = F)
pnd8 &lt;- cd$Stage == &quot;PND8&quot;
sePND8 &lt;- se[,pnd8]
sePND8_exon &lt;- sePND8
sePND8_exon &lt;- diffSpliceWrapper(sePND8_exon, design = ~Batch+Group)</code></pre>
<pre><code>Testing coefficient GroupMSUS</code></pre>
<pre><code>Total number of exons:  316430 
Total number of genes:  22903 
Number of genes with 1 exon:  5745 
Mean number of exons in a gene:  14 
Max number of exons in a gene:  418 </code></pre>
<pre class="r"><code>se_apa &lt;- readRDS(&quot;C:/Users/chabb/Desktop/ETHZ/3. Jahr/6. Semester/Blockkurs/2. Epigenetic mechanism/Project/project6_bc2021/data/counts_bins_apa.rds&quot;)

cd2 &lt;- data.frame(colData(se_apa))
pnd8_2 &lt;- cd2$Stage == &quot;PND8&quot;
sePND8UTR &lt;- se_apa[,pnd8_2]
sePND8_utr &lt;- diffSpliceWrapper(sePND8UTR, design = ~Batch+Group, 
                excludeTypes = c(&quot;CDS&quot;,&quot;non-coding&quot;))</code></pre>
<pre><code>Testing coefficient GroupMSUS</code></pre>
<pre><code>Total number of exons:  316430 
Total number of genes:  22903 
Number of genes with 1 exon:  5745 
Mean number of exons in a gene:  14 
Max number of exons in a gene:  418 </code></pre>
<p>#P-Value Histogram</p>
<pre class="r"><code>hist.plot_exon &lt;- hist(rowRanges(sePND8_exon)$bin.p.value)</code></pre>
<p><img src="figure/3_Processed_plot_PND8_JC.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>hist.plot_utr &lt;- hist(rowRanges(sePND8_utr)$bin.p.value)</code></pre>
<p><img src="figure/3_Processed_plot_PND8_JC.Rmd/unnamed-chunk-3-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>#MA Plot</p>
<pre class="r"><code>#Exon

#data preparation
maData &lt;- data.frame(baseMean = rowData(sePND8_exon)$meanLogCPM, log2FoldChange = rowData(sePND8_exon)$coefficient, padj = rowData(sePND8_exon)$bin.FDR)

#plot
ggmaplot(maData, 
   fdr = 0.05, fc = 2, size = 0.4,
    xlab = &quot;Log2 mean expression&quot;,
    ylab = &quot;Log2 fold change&quot;,
    top = 3,
    font.label = c(&quot;bold&quot;, 11), label.rectangle = TRUE,
    font.legend = &quot;bold&quot;,
    font.main = &quot;bold&quot;
  
)</code></pre>
<p><img src="figure/3_Processed_plot_PND8_JC.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#UTR

#data preparation
maData_utr &lt;- data.frame(baseMean = rowData(sePND8_utr)$meanLogCPM, log2FoldChange = rowData(sePND8_utr)$coefficient, padj = rowData(sePND8_utr)$bin.FDR)

#plot
ggmaplot(maData_utr, 
   fdr = 0.05, fc = 2, size = 0.4,
    xlab = &quot;Log2 mean expression&quot;,
    ylab = &quot;Log2 fold change&quot;,
    top = 3,
    font.label = c(&quot;bold&quot;, 11), label.rectangle = TRUE,
    font.legend = &quot;bold&quot;,
    font.main = &quot;bold&quot;
  
)</code></pre>
<p><img src="figure/3_Processed_plot_PND8_JC.Rmd/unnamed-chunk-4-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>#Volcano Plot (exon analysis)</p>
<pre class="r"><code>#Exon

EnhancedVolcano(rowData(sePND8_exon),
    lab = rownames(rowData(sePND8_exon)),
    x = &#39;coefficient&#39;,
    y = &#39;bin.FDR&#39;,
        )</code></pre>
<p><img src="figure/3_Processed_plot_PND8_JC.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#UTR

EnhancedVolcano(rowData(sePND8_utr),
    lab = rownames(rowData(sePND8_utr)),
    x = &#39;coefficient&#39;,
    y = &#39;bin.FDR&#39;,
        )</code></pre>
<p><img src="figure/3_Processed_plot_PND8_JC.Rmd/unnamed-chunk-5-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>#Heatmap</p>
<pre class="r"><code>#Exon

significant &lt;- rowRanges(sePND8_exon)$bin.p.value &lt;= 0.01
sePND8_significant &lt;- sePND8_exon[significant,]

sechm(sePND8_significant, genes=rownames(assay(sePND8_significant)),do.scale = TRUE)</code></pre>
<pre><code>Using assay logcpm</code></pre>
<pre><code>`use_raster` is automatically set to TRUE for a matrix with more than
2000 rows. You can control `use_raster` argument by explicitly setting
TRUE/FALSE to it.

Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<pre><code>&#39;magick&#39; package is suggested to install to give better rasterization.

Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<p><img src="figure/3_Processed_plot_PND8_JC.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#UTR

significant_utr &lt;- rowRanges(sePND8_utr)$bin.p.value &lt;= 0.01
sePND8_significant_utr &lt;- sePND8_utr[significant_utr,]

sechm(sePND8_significant_utr, genes=rownames(assay(sePND8_significant)),
      do.scale = TRUE)</code></pre>
<pre><code>Using assay logcpm</code></pre>
<pre><code>`use_raster` is automatically set to TRUE for a matrix with more than
2000 rows. You can control `use_raster` argument by explicitly setting
TRUE/FALSE to it.

Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<pre><code>&#39;magick&#39; package is suggested to install to give better rasterization.

Set `ht_opt$message = FALSE` to turn off this message.</code></pre>
<p><img src="figure/3_Processed_plot_PND8_JC.Rmd/unnamed-chunk-6-2.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.4 (2021-02-15)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=French_Switzerland.1252  LC_CTYPE=French_Switzerland.1252   
[3] LC_MONETARY=French_Switzerland.1252 LC_NUMERIC=C                       
[5] LC_TIME=French_Switzerland.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] EnhancedVolcano_1.8.0       ggrepel_0.9.1              
 [3] SEtools_1.4.0               ggpubr_0.4.0               
 [5] ensembldb_2.14.0            AnnotationFilter_1.14.0    
 [7] GenomicFeatures_1.42.2      AnnotationDbi_1.52.0       
 [9] edgeR_3.32.1                limma_3.46.0               
[11] reshape2_1.4.4              ggplot2_3.3.3              
[13] dplyr_1.0.5                 SummarizedExperiment_1.20.0
[15] Biobase_2.50.0              GenomicRanges_1.42.0       
[17] GenomeInfoDb_1.26.4         IRanges_2.24.1             
[19] S4Vectors_0.28.1            BiocGenerics_0.36.0        
[21] MatrixGenerics_1.2.1        matrixStats_0.58.0         
[23] diffUTR_0.99.24             workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] utf8_1.2.1               tidyselect_1.1.0         RSQLite_2.2.4           
  [4] grid_4.0.4               TSP_1.1-10               BiocParallel_1.24.1     
  [7] Rtsne_0.15               munsell_0.5.0            codetools_0.2-18        
 [10] statmod_1.4.35           withr_2.4.1              colorspace_2.0-0        
 [13] ggalt_0.4.0              highr_0.8                knitr_1.31              
 [16] ggsignif_0.6.1           Rttf2pt1_1.3.8           Rsubread_2.4.3          
 [19] labeling_0.4.2           git2r_0.28.0             GenomeInfoDbData_1.2.4  
 [22] hwriter_1.3.2            farver_2.1.0             bit64_4.0.5             
 [25] pheatmap_1.0.12          rprojroot_2.0.2          vctrs_0.3.6             
 [28] generics_0.1.0           xfun_0.22                BiocFileCache_1.14.0    
 [31] R6_2.5.0                 ggbeeswarm_0.6.0         clue_0.3-58             
 [34] seriation_1.2-9          locfit_1.5-9.4           bitops_1.0-6            
 [37] cachem_1.0.4             DelayedArray_0.16.2      assertthat_0.2.1        
 [40] promises_1.2.0.1         scales_1.1.1             beeswarm_0.3.1          
 [43] gtable_0.3.0             ash_1.0-15               Cairo_1.5-12.2          
 [46] rlang_0.4.10             genefilter_1.72.1        GlobalOptions_0.1.2     
 [49] splines_4.0.4            extrafontdb_1.0          rtracklayer_1.49.5      
 [52] rstatix_0.7.0            lazyeval_0.2.2           broom_0.7.5             
 [55] yaml_2.2.1               abind_1.4-5              backports_1.2.1         
 [58] httpuv_1.5.5             extrafont_0.17           tools_4.0.4             
 [61] ellipsis_0.3.1           jquerylib_0.1.3          RColorBrewer_1.1-2      
 [64] Rcpp_1.0.6               plyr_1.8.6               progress_1.2.2          
 [67] zlibbioc_1.36.0          purrr_0.3.4              RCurl_1.98-1.3          
 [70] prettyunits_1.1.1        openssl_1.4.3            GetoptLong_1.0.5        
 [73] haven_2.3.1              cluster_2.1.1            fs_1.5.0                
 [76] magrittr_2.0.1           data.table_1.14.0        openxlsx_4.2.3          
 [79] circlize_0.4.12          ProtGenerics_1.22.0      randomcoloR_1.1.0.1     
 [82] hms_1.0.0                evaluate_0.14            xtable_1.8-4            
 [85] XML_3.99-0.6             rio_0.5.26               readxl_1.3.1            
 [88] shape_1.4.5              compiler_4.0.4           biomaRt_2.46.3          
 [91] tibble_3.1.0             maps_3.3.0               KernSmooth_2.23-18      
 [94] V8_3.4.0                 crayon_1.4.1             htmltools_0.5.1.1       
 [97] later_1.1.0.1            tidyr_1.1.3              geneplotter_1.68.0      
[100] DBI_1.1.1                proj4_1.0-10.1           dbplyr_2.1.0            
[103] ComplexHeatmap_2.6.2     MASS_7.3-53.1            rappdirs_0.3.3          
[106] Matrix_1.3-2             car_3.0-10               forcats_0.5.1           
[109] pkgconfig_2.0.3          GenomicAlignments_1.26.0 registry_0.5-1          
[112] foreign_0.8-81           xml2_1.3.2               foreach_1.5.1           
[115] annotate_1.68.0          vipor_0.4.5              bslib_0.2.4             
[118] XVector_0.30.0           DEXSeq_1.36.0            stringr_1.4.0           
[121] digest_0.6.27            Biostrings_2.58.0        rmarkdown_2.7           
[124] cellranger_1.1.0         curl_4.3                 Rsamtools_2.6.0         
[127] rjson_0.2.20             lifecycle_1.0.0          jsonlite_1.7.2          
[130] carData_3.0-4            askpass_1.1              fansi_0.4.2             
[133] pillar_1.5.1             lattice_0.20-41          ggrastr_0.2.3           
[136] fastmap_1.1.0            httr_1.4.2               survival_3.2-10         
[139] glue_1.4.2               zip_2.1.1                png_0.1-7               
[142] iterators_1.0.13         bit_4.0.4                stringi_1.5.3           
[145] sass_0.3.1               blob_1.2.1               DESeq2_1.30.1           
[148] memoise_2.0.0           </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
